File: Tests/CSharp/IncludeDirectories/IncludeDirectories.cs
// **********************************************************************
//
// Copyright (c) 2009-2016 ZeroC, Inc. All rights reserved.
//
// **********************************************************************

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Test
{
    public class IncludeDirectories
    {
        public static bool includeDirectories()
        {
            Module2.Derived d = new Module2.Derived();
            return true;
        }
    }
}

File: Tests/Cpp/IncludeDirectories/IncludeDirectories.cpp
// **********************************************************************
//
// Copyright (c) 2009-2016 ZeroC, Inc. All rights reserved.
//
// **********************************************************************

#include <Ice\Ice.h>
#include "IncludeDirectories.h"
#include <Module2.h>

bool Test::includeDirectories()
{
	Module2::DerivedPtr d = new Module2::Derived();
	return true;
}